src/HtmlUtils.tsx

```diff
@@ -399,6 +399,7 @@ export function bodyToHtml(content: IContent, highlights: Optional<string[]>, op
         "mx_EventTile_body": true,
         "mx_EventTile_bigEmoji": emojiBody,
         "markdown-body": isHtmlMessage && !emojiBody,
+        "translate": true,
```

Comment:
The entire application currently has translation disabled using the notranslate class placed in the div#matrixchat element.
The translate class overrides the notranslate class that is in this element's ancestor.

Reply:
please edit the file and add a comment starting `//` or `/*`.

Reply:
```suggestion
        // Override the global `notranslate` class set by the top-level `matrixchat` div.
        "translate": true,
```